hls4ml/converters/keras/core.py

```diff
@@ -62,6 +62,10 @@ def parse_activation_layer(keras_layer, input_names, input_shapes, data_reader):
 
     if layer['class_name'] != 'Activation':
         layer['activation'] = layer['class_name']
+
+    if layer['activation'] == 'elu':
+        layer['class_name'] = 'ELU'  # always use ELU type for elu, even if passed as activation
```

Comment:
Do we need this also for the other converters?

Reply:
For oneAPI we probably would need it everywhere. If I remember correctly, this has to do with oneAPI kernels not liking function overloading, but I would have to check. It may be worth enabling the pytorch tests that would exercise this.

Reply:
Or maybe not. If I remember correctly, for Keras 2 API you could either use an ELU layer (https://keras.io/2.17/api/layers/activation_layers/elu/) or an Activation layer (https://keras.io/2.17/api/layers/core_layers/activation/) with an elu activation function, and this was so that they always became the same node type in hls4ml. I didn't think hls4ml needed the two different types of ELUs.

Reply:
I think I need to add 'oneAPI' to `test_pytorch_api.py` to test for this. I'll try that out.

Reply:
Actually, the activations succeed without a problem. It is MaxPooling1D with pad of 1 that hangs (while avg pooling works just fine).

Reply:
This pooling with padding issue was fixed by [8c827b8](https://github.com/fastmachinelearning/hls4ml/pull/955/commits/8c827b83af689d0efb6a1292efdaed888703e42c)